tests/typings/webdriverio-applitools/applitools.ts

```diff
@@ -0,0 +1,20 @@
+const config: WebdriverIO.Config = {
+    applitoolsKey: '',
+    applitoolsServerUrl: '',
+    applitools: {
+        viewport: {
+            width: 1,
+            height: 1
+        }
+    }
+}
+
+browser.takeSnapshot('title');
```

Comment:
Can you move it please to async function and add await?

Reply:
Done